src/client/pages/profile/profile.jsx

```diff
@@ -71,6 +73,10 @@ class Profile extends PureComponent {
     }
   }
 
+  cancel = () => {
+    history.push(routes.home.path);

```

Comment:
I think the `cancel` button should return all filled fields to their original state, not redirect to another page.

What makes you think there should be a redirect? 🤔